pydocstringformatter/run.py

```diff
@@ -6,22 +6,27 @@
 import sys
 import tokenize
 from pathlib import Path
-from typing import List, Union
+from typing import List, Optional, Union
 
-from pydocstringformatter import __version__, formatting, utils
+from pydocstringformatter import __version__, utils
+from pydocstringformatter.formatting import FORMATTERS, Formatter
 
 
 class _Run:
     """Main class that represent a run of the program."""
 
-    def __init__(self, argv: Union[List[str], None]) -> None:
+    def __init__(
+        self, argv: Union[List[str], None], formatters: Optional[List[Formatter]] = None
```

Comment:
Allowing additional `formatters` should probably go in its own PR

Reply:
I started creating another MR, but it's embedded with the way we changed the formatter to become optional, thus hard to cherry-pick.